### ISSUE TITLE
disable root to use apps with root detection

### ISSUE THREAD

**Issue by baptx** [anbox/anbox#1453]
I noticed it is not possible to sign up with Snapchat when using Anbox. I read the issue may come from root detection (https://www.reddit.com/r/xposed/comments/5w6ihe/help_unable_to_sign_up_on_snapchat_even_when/).
Is it possible disable root in Anbox? This would allow to use apps with root detection, without having to disassemble or decompile an app to disable root detection.

**Comment by necessarily-equal** [comment]
In my opinion, we should just remove the `su` binary. It's broken anyway. I think the usual method to get root is to use the `anbox-shell.sh` script.

related: since yesterday we remove it from the image on postmarketOS' side: https://gitlab.com/postmarketOS/pmaports/-/merge_requests/1230/diffs#d434bcb20e53cd250c27f9e7b375b2f83f904746_48_49

**Comment by morphis** [comment]
Yeah, removing su should be the right way. The next set of images will be build with `user` instead of `userdebug`.

**Comment by baptx** [comment]
@necessarily-equal @morphis thanks, I removed the file `/system/xbin/su` from android.img in the snap package using these commands: https://github.com/anbox/anbox/issues/1368#issuecomment-622564745
However I still cannot login with Snapchat (without VPN). Note that removing the su binary may not be enough to hide that we can get root with Anbox. Some Android apps are using a root detection library like https://github.com/scottyab/rootbeer which is checking a lot of things. Is there something else that needs to be removed or hidden in Anbox to pass a root detection test?
By the way, good to see that Anbox is supported in PostmarketOS, this may be an alternative to SailfishOS for people using Jolla smartphone (https://github.com/anbox/anbox/issues/678).
